proto/rill/admin/v1/api.proto

```diff
@@ -1839,14 +1839,20 @@ message GetReportMetaRequest {
   string project_id = 1;
   string branch = 2;
   string report = 3;
-  map<string, string> annotations = 4;
-  google.protobuf.Timestamp execution_time = 5;
+  // 4 was previously used for annotations
```

Comment:
Can remove this comment, just put a `reserved` statement on the last line of the message definition (if people need to know why, they can look in the Git history).